src/main/com/example010/radek/v1/IBaseFighter.java

```diff
@@ -0,0 +1,10 @@
+package main.com.example010.radek.v1;
+
+public interface IBaseFighter {

```

Comment:
je to sice pekne, ale neuc se to :D ... Icko pryc

Reply:
Hotovo :D